src/providers/linkedin.ts

```diff
@@ -59,14 +65,30 @@ export class LinkedIn implements OAuth2Provider {
 	}
 }
 
-interface TokenResponseBody {
+interface AuthorizationCodeResponseBody {
+	id_token: string;
+	access_token: string;
+	expires_in: number;
+	refresh_token?: string; // available only if your application is authorized for programmatic refresh tokens
+	refresh_token_expires_in?: number;
+}
+
+interface RefreshTokenResponseBody {
```

Comment:
Is the ID token not available when you refresh the token?

Reply:
Right. The ID token is not available in the response of the tokenEndpoint. See the link in my initial comment.
It is the similar situation as in google.ts where the tokenEndpoint does not return ID token.